### PR TITLE
Updated Contribution.md file

### PR DESCRIPTION
**Pull request by shishir-intelli**
https://github.com/apigee/apigee-edge-drupal/issues/998
Updated the Contribution.md file and renamed Apigee Edge to Apigee.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,20 +30,20 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
...
 * Add changes to the code. If you implement new features, add new
 tests to cover the implemented functionality. If you modify existing features, update related tests.
 * Push your changes to your repo's branch.
-* Wait until all CircleCI test jobs finish and _pass_. (If any of them fails
-restart them once or twice. They may have failed due to an API communication error. You can
+* Wait until all Github Actions/CircleCI test jobs finish and _pass_. (If any of them fails
+restart them once or twice. They may have failed due to an API communication error or caching issue. You can
 identify these type of issues from logs.)
-* Create [new pull request](https://github.com/apigee/apigee-edge-drupal/pull/new/8.x-1.x). CircleCI will
+* Create [new pull request](https://github.com/apigee/apigee-edge-drupal/pull/new/8.x-1.x). Github Actions/CircleCI will
```
Comment (divya-intelli) [inline CONTRIBUTING.md]: Change the default branch to 3.x

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,20 +30,20 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
...
 * Install the module from for your fork instead of Drupal.org on your local. (See below.)
 
 ## For daily work
 * Create a new branch in your fork repository. It is best to name your branch something descriptive, ex.: issue-12.
 * Add changes to the code. If you implement new features, add new
 tests to cover the implemented functionality. If you modify existing features, update related tests.
 * Push your changes to your repo's branch.
-* Wait until all CircleCI test jobs finish and _pass_. (If any of them fails
-restart them once or twice. They may have failed due to an API communication error. You can
+* Wait until all Github Actions/CircleCI test jobs finish and _pass_. (If any of them fails
```
Comment (divya-intelli) [inline CONTRIBUTING.md]: Remove the reference to CircleCi.

File: CONTRIBUTING.md (hunk 1 above)
Comment (shishir-intelli) [reply CONTRIBUTING.md]: Done

**Review by kedarkhaire — APPROVED** [review]
LGTM

**Pull request merged**